README.md

```diff
@@ -0,0 +1,152 @@
+<head>
+  <meta name="Momento Laravel cache driver example" content="Taggable Momento serverless cache driver example for Laravel">
+</head>
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+# Momento Laravel Example
+In this repo, you will see an example of how to integrate a Momento cache into your Laravel app, instead of a Redis or Memcached.
+What's the benefits, you ask? You don't have to worry about Redis/Memcached nodes!
+Intrigued? Keep on reading!
+
+## Run the app via Docker
+
+Build a Docker image for the app:
+
+Coming soon.
+
+## cURL commands
+```bash
+curl http://127.0.0.1:8000/api/weather/<your-favorite-city>
+
+Example:
+curl http://127.0.0.1:8000/api/weather/denver
+```
+
+or
+
+```bash
+curl http://127.0.0.1:8000/api/weather/<your-favorite-zipcode>/<country-code-such-as-us>
+
+Example:
+curl http://127.0.0.1:8000/api/weather/98101/us
+```
+
+or
+
+```bash
+curl http://127.0.0.1:8000/api/weather/id/<your-favorite-city-id>
+
+City id can be found here: http://bulk.openweathermap.org/sample/
+
+Example:
+curl http://127.0.0.1:8000/api/weather/id/833
+```
+
+## Exploring the Momento Cache Integration
```

Comment:
We should talk about whether or not we want to include this section with this sample code. If there's a Laravel driver available, I can't imagine many people would want to ignore that and use our client directly. If we do want to include this info, I'd argue it should probably go in the `laravel-cache` repository README instead of this one.